flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/batch/sql/CorrelateTest.scala

```diff
@@ -41,7 +41,8 @@ class CorrelateTest extends TableTestBase {
         "DataSetCorrelate",
         batchTableNode(0),
         term("invocation", "func1($cor0.c)"),
-        term("function", func1.getClass.getCanonicalName),
+        term("correlate", s"table(func1($$cor0.c))"),
+        term("select", "a,b,c,f0"),
```

Comment:
please change to `term("select", "a", "b", "c", "f0"),` (i.e., use separate strings for the field names) for consistency